agent/firefoxextensions.ps1

```diff
@@ -86,14 +86,10 @@ if ((test-path 'C:\Program Files (x86)\Mozilla Firefox\firefox.exe') -or (test-p
                 } 
                       
             } 
-            else 
-  {
-    $xml += "<BROWSEREXTENSIONS />"
-  }
         } 
     } 
- 
-  
+} else {
```

Comment:
@Lea9250 ,

There is one use case that doesn't work currently, if you install firefox and only have built-in / system extensions it will return an empty output and generate an error on the agent log.

This else condition is not taking all the use case into account. If you replace this by : 
```
if($xml -eq $null) {
	$xml = "<BROWSEREXTENSIONS/>"
}
```

That should fix the issue.

Regards,
Charlene